src/nn/ops/reduce.ts

```diff
@@ -15,10 +15,8 @@ abstract class Reduce extends SingleOutputOperation {
     utils.validateOperand(input);
     this.input_ = input;
     if (options.axes !== undefined) {
-      utils.assert(
-          utils.isIntegerArray(options.axes) &&
-              options.axes.every(v => v > 0 || v === -1),
-          'The starts parameter is invalid.');
+      utils.assert(utils.isIntegerArray(options.axes),
+          'The axes parameter is invalid.');
```

Comment:
Could you please follow the [spec](https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-reduce) to validate the `axes`. 

> axes: a sequence of long. The dimensions to reduce where -1 means the last dimension. If not present, all dimensions are reduced.